cmake/inputs/DownloadPrebuiltTileDB.cmake

```diff
@@ -74,7 +74,7 @@ endfunction()
 
 function(detect_artifact_name OUT_VAR)
         if (WIN32) # Windows
-                SET(${OUT_VAR} TILEDB_WINDOWS_X86_64 PARENT_SCOPE)
+                SET(${OUT_VAR} TILEDB_WINDOWS-X86_64 PARENT_SCOPE)
```

Comment:
We should fail on other architectures. Same in Linux.

Reply:
I think that CMAKE can only define these three variables: WIN32/APPLE/LINUX